docker/docker-compose.yml

```diff
@@ -179,6 +183,20 @@ services:
       - 5000:5000
     volumes:
       - registry-data:/var/lib/registry
+  minio:
+    image: quay.io/minio/minio:RELEASE.2023-01-25T00-19-54Z
```

Comment:
That's just the versioning scheme that MinIO uses

Reply:
I guess I'm suggesting that I'd rather just use the `latest` tag and if that breaks our stuff one day we'll fix it, rather than pinning to this super specific version (which is already not the newest release) and then likely staying on it for way too long.

Reply:
Fixed